apps/tree-shaking-test-app/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import { barCode } from 'testlibraryfortreeshaking';
+//import { PrintTestMessage } from 'testlibraryfortreeshaking/app';
+import { geoLocation } from '@microsoft/teams-js';
+//import { app } from '@microsoft/teams-js';
```

Comment:
Do we want to keep these commented out? If so can we have an added comment explaining why

Reply:
Oh, yeah, I forgot about that file. The whole tree-shaking test app isn't actually doing anything. I was just using it for testing, and wanted to check it in for use later. I'll go ahead and remove the comments so it looks nicer.